src/tooltips.tsx

```diff
@@ -201,7 +245,13 @@ class TooltipComponent extends React.Component<{}, TooltipComponentState> {
 	}
 
 	hide(): void {
-		this.setState({ show: false });
+		this.setState({
```

Comment:
We should also probably disconnect & de-reference the observer, since we no longer care about the mutations on the target and let the target be GCed if removed from the DOM.
```suggestion
		if (this.observer) {
			this.observer.disconnect();
			this.observer = undefined;
		}
		this.setState({
```
^ GH does a bad diff on suggestions that just prepend some code :(

Reply:
Good idea